From 32d7d5c02d4133892d14f327c56eeab2465d8b49 Mon Sep 17 00:00:00 2001
From: duelu <luyizhou1@huawei.com>
Date: Thu, 21 May 2026 03:52:46 +0800
Subject: [PATCH] urma: change the bazel compile result from liburma_udma.so to
 liburma-udma.so

(cherry picked from commit 977f1779f364c1f3db8b5125bb4adad22bcad19b)
---
 src/urma/BUILD.bazel | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/urma/BUILD.bazel b/src/urma/BUILD.bazel
index 646cd312..d1aa81d9 100644
--- a/src/urma/BUILD.bazel
+++ b/src/urma/BUILD.bazel
@@ -112,7 +112,7 @@ alias(
 )
 
 cc_library(
-    name = "urma_udma",
+    name = "urma-udma",
     srcs = select({
         ":build_udma_enabled": glob(["hw/udma/*.c"]),
         "//conditions:default": [],
@@ -142,5 +142,5 @@ cc_library(
 
 alias(
     name = "hw_udma",
-    actual = ":urma_udma",
+    actual = ":urma-udma",
 )
-- 
2.43.0

